hushline/__init__.py

```diff
@@ -61,6 +61,9 @@ def create_app() -> Flask:
     db.init_app(app)
     Migrate(app, db)
 
+    with app.app_context():
+        db.create_all()
```

Comment:
is this needed? Don't we create the tables already?

Reply:
Addressed by (https://github.com/scidsg/hushline/pull/573/commits/e9ca02df0fad9c3aa1e25102f599a0520d0340d7). Commented on also in (https://github.com/scidsg/hushline/pull/573#discussion_r1761779499).